### ISSUE TITLE
Build list not displaying properly

### ISSUE THREAD

**Issue by dogtopus** [lavanoid/lavanoid.github.io#4]
![screenshot from 2016-02-28 02-16-04](https://cloud.githubusercontent.com/assets/3405234/13377811/f5bed92c-ddc1-11e5-97e7-3c505f60915f.png)

**Comment by lavanoid** [comment]
Yeah, the dude who gave me the VPS f***ed everything up for me. PHP is broke and I don't have a proper internet connection or time to fix it. Sorry :/

I'll fix it whenever I am able to do so.
